hail/python/hailtop/hailctl/dataproc/cli.py

```diff
@@ -113,7 +113,12 @@ def start(
     properties: Ann[Optional[str], Opt(help='Additional configuration properties for the cluster.')] = None,
     metadata: Ann[Optional[str], Opt(help='Comma-separated list of metadata to add: KEY1=VALUE1,KEY2=VALUE2')] = None,
     packages: Ann[
-        Optional[str], Opt(help='Comma-separated list of Python packages to be installed on the master node.')
+        Optional[str],
+        Opt(
+            '--packages',
+            '--pkgs',
```

Comment:
I gotta say, I like how this is a simple, straightforward, and self-evident fix.